packages/lib-classifier/src/plugins/tasks/SurveyTask/components/components/Chooser/components/CharacteristicsFilter/ClearFilters/ClearFilters.js

```diff
@@ -0,0 +1,83 @@
+import { Box } from 'grommet'
+import { Clear } from 'grommet-icons'
+import PropTypes from 'prop-types'
+import React from 'react'
+import { PlainButton, SpacedText } from '@zooniverse/react-components'
+
+import counterpart from 'counterpart'
+import en from '../locales/en'
+
+counterpart.registerTranslations('en', en)
+
+function Label () {
+  return (
+    <Box
+      align='center'
+      direction='row'
+      gap='xxsmall'
+    >
+      <Clear
+        color={{
+          dark: 'light-3',
+          light: 'dark-5'
+        }}
+        size='small'
+      />
+      <SpacedText>
+        {counterpart('CharacteristicsFilter.clearFilters')}
+      </SpacedText>
+    </Box>
+  )
+}
+
+export default function ClearFilters (props) {
+  const {
+    handleFilter,
+    showingChoices,
+    totalChoices
+  } = props
+
+  return (
+    <Box
+      align='center'
+      direction='row'
+      fill='horizontal'
+      gap='xxsmall'
+      justify='center'
+      pad={{ top: 'xsmall' }}
+    >
+      <SpacedText>
+        {counterpart('CharacteristicsFilter.showing', { showing: showingChoices, total: totalChoices })}
+      </SpacedText>
+      <PlainButton
+        color={{
+          dark: 'light-3',
+          light: 'dark-5'
+        }}
+        disabled={showingChoices === totalChoices}
+        icon={<Clear
+          color={{
+            dark: 'light-3',
+            light: 'dark-5'
+          }}
+          size='small'
+        />}
+        label={counterpart('CharacteristicsFilter.clearFilters')}
+        onClick={() => handleFilter()}
+        plain
+      />
```

Comment:
I misremembered the prop for the text of the label, sorry!
```suggestion
      <PlainButton
        disabled={showingChoices === totalChoices}
        icon={<Clear />}
        onClick={() => handleFilter()}
        text={counterpart('CharacteristicsFilter.clearFilters')}
      />
```
A couple other tweaks here too, removed `plain` and the styles from the icon since it's also redundant with the component and removed the color so it reuses the standard colors applied to the `PlainButton` for consistency.